src/ExpressiveInstaller/config.php

```diff
@@ -11,13 +11,22 @@
         'zendframework/zend-expressive-twigrenderer'     => '^1.2.1',
         'zendframework/zend-expressive-zendrouter'       => '^2.0',
         'zendframework/zend-expressive-zendviewrenderer' => '^1.2.1',
-        'zendframework/zend-servicemanager'              => '^2.7.3 || ^3.0',
+        'zendframework/zend-servicemanager'              => '^2.7.3 || ^3.2',
     ],
 
     'require-dev' => [
         'filp/whoops'
     ],
 
+    'application' => [
+        'flat' => [
+            '/Resources/src/ConfigProvider.flat.php' => '/src/App/ConfigProvider.php',
+        ],
+        'modular' => [
+            '/Resources/src/ConfigProvider.modular.php' => '/src/App/src/ConfigProvider.php',
```

Comment:
Use something like this?

    'application' => [
        'copy-files' => [
            '/Resources/src/ConfigProvider.flat.php' => [
                'flat'    => '/src/App/ConfigProvider.php',
            ],
            '/Resources/src/ConfigProvider.modular.php' => [
                'modular' => '/src/App/src/ConfigProvider.php',
            ],
        ],
    ],

It's more inline with other questions.